styles.css

```diff
@@ -0,0 +1,521 @@
+@import url('https://fonts.googleapis.com/css2?family=Lato:wght@400;700&display=swap');
+
+@font-face {
+  src: url(font/CocogooseProItalic-trial.ttf);
+  font-family: Cocogoose Pro;
+}
+
```

Comment:
- [ ] Your mobile design overflows kindly fix this 🙏 

![20220728-171613_capture](https://user-images.githubusercontent.com/74814780/181597493-7498c502-6dfe-4007-bc54-519654942da7.gif)